java2023/src/main/java/com/practica/java2023/model/Event.java

```diff
@@ -0,0 +1,95 @@
+package com.practica.java2023.model;
+
+import jakarta.persistence.*;
+
+import java.io.Serializable;
+import java.time.LocalDateTime;
+
+@Entity
+@Table(name = "Events")
+public class Event implements Serializable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
```

Comment:
You don't need to explicitly put GenerationType.AUTO, as this is the default value. If you chose another one, you need to indicate it.